Buildings/Controls/OBC/ASHRAE/G36/ThermalZones/ControlLoops.mo

```diff
@@ -53,15 +53,20 @@ block ControlLoops "Heating and cooling control loops"
     final unit="1") "Heating control signal"
     annotation (Placement(transformation(extent={{160,-90},{200,-50}}),
       iconTransformation(extent={{100,-80},{140,-40}})));
-
-protected
   Buildings.Controls.OBC.CDL.Continuous.PIDWithReset cooCon(
```

Comment:
Shouldn't they rather be called `conCoo` and `conHea` and usually we start controllers with `con`.